glean-core/ios/GleanTests/Scheduler/MetricsPingSchedulerTests.swift

```diff
@@ -259,18 +259,11 @@ class MetricsPingSchedulerTests: XCTestCase {
         // Set up the interception of the ping for inspection
         let host = URL(string: Configuration.Constants.defaultTelemetryEndpoint)!.host!
         stub(condition: isHost(host)) { data in
-            let body = (data as NSURLRequest).ohhttpStubs_HTTPBody()
-            let json = try! JSONSerialization.jsonObject(with: body!, options: []) as? [String: Any]
-            XCTAssert(json != nil)
-            let metrics = json?["metrics"] as? [String: Any]
-            let strings = metrics?["string"] as? [String: Any]
-
-            // Ensure there is only the expected metric
-            XCTAssertEqual(1, strings?.count, "Must contain only the expected metric")
+            let docType = (data as NSURLRequest).url!.path
```

Comment:
In other tests we are a bit more explicit, checking the right fragment of the path:

```swift
request.url?.path.split(separator: "/")[2]
```